### ISSUE TITLE
Issue with using import in content scripts

### ISSUE THREAD

**Issue by bryce13950** [fregante/browser-extension-template#65]
The default template is throwing this error `SyntaxError: import declarations may only appear at top level of a module` when attempting to access a page that should trigger the content.js script. Upon searching a bit, I found that the issue was a limitation of the files that can be included through content_scripts due to needing the type="module" attribute needing to be set on the script https://stackoverflow.com/questions/48104433/how-to-import-es6-modules-in-content-script-for-chrome-extension. Is there a version of this template that has this working properly? Does a transpolar need to be configured in order to maintain syntax when dealing with content?

**Comment by fregante** [comment]
Probably due to the last updates. Can you try running this in your repo?

```js
git revert fe6d903c2b4da6d30ae1d15dd00dd860965e8103
npm install
npm run build
```

**Comment by bryce13950** [comment]
Sorry about the late reply here. I did try on the version before with the dependency upgrade, and I was still getting that error.

<img width="1118" alt="image" src="https://user-images.githubusercontent.com/6487928/159099000-a60544a1-fe6d-445f-a013-ffdb6e14d2ae.png">

I am able to get feedback from the content.js script by removing the import and adding an alert or log. Is there a way to get that script to be marked as a module?

**Comment by fregante** [comment]
I don't have time to investigate this but it's more likely that you're still using more recent versions of Parcel. Make sure that it's the correct one by downloading the package-lock file from the repo and then installing that again

**Comment by fregante** [comment]
@bryce13950 are you loading the files in the `distribution` folder or in `source`? I tried it myself and it works correctly.

**Comment by bryce13950** [comment]
Apologies! I had some more time to mess with this today, and I realized almost immediately what my issue was. When I downloaded the project, I copied the contents of the project into my repo, and I did not notice the hidden files. Upon adding those, all issues resolved. I appreciate the patience, and hopefully at the very least if someone else has this issue they can see this, and know to double check that those hidden files are in their repo.
